app/controllers/welcome_controller.rb

```diff
@@ -5,7 +5,6 @@
 class WelcomeController < ApplicationController
   def index
     @shows = CSV.foreach('netflix_titles.csv', headers: true, header_converters: :symbol)
-                .take((params[:limit] || 10).to_i)
-
+                .take((params[:limit] || 12).to_i)
```

Comment:
also would be great if you could move this csv parsing into separate location like a helper or a concern